packages/shared/locales/en.json

```diff
@@ -947,8 +947,7 @@
                     "network": "Network",
                     "bech32Hrp": "Bech32 HRP",
                     "tokenSupply": "Token supply",
-                    "version": "Version",
-                    "minPowScore": "Min PoW Score"
+                    "version": "Version"
```

Comment:
There are more references to clean in this file like `views.settings.localProofOfWork` or `general.transferPow`